src/videoserver/lib/utils.py

```diff
@@ -99,6 +99,15 @@ def _handle_doc(doc):
                     _external=True
                 )
 
+            if doc['processing']['thumbnail_preview']:
+                doc['thumbnails']['preview'] = {
+                    "url": url_for(
+                        'projects.get_raw_preview_thumbnail',
+                        project_id=doc['_id'],
+                        _external=True
+                    ),
+                }
+
```

Comment:
This change return a predict url when task preview thumbnails is processing.
Because after we call api to push a task to get preview thumbnail, we must ensure the url must be update in Superdesk.
If we use a polling from client to check and update data, maybe user close the browser and we missing data. Even when we use notification in client, it also is not safe.
If a polling from server Superdesk, it is too much complicated and make server is heavier, But it still have case when Superdesk is down, we also will missing data.
So we should return a predict correct url, after call task success and update database immediately.

Reply:
I think that url for a raw preview thumbnail is always the same.